Core/src/com/infiniteautomation/mango/spring/components/TokenAuthenticationService.java

```diff
@@ -51,7 +52,12 @@ public final class TokenAuthenticationService extends JwtSignerVerifier<User> {
     public TokenAuthenticationService(
             PermissionService permissionService,
             UsersService usersService,
-            ApplicationContext context, RunAs runAs, UserDao userDao) {
+            ApplicationContext context,
+            RunAs runAs,
+            UserDao userDao,
+            Clock clock) {
```

Comment:
Modified API in a patch release, might be safer to not backport the tests (this change was for testing only).

Reply:
Ok, undoing it.